packages/backend/src/build-disk-image.ts

```diff
@@ -130,6 +130,11 @@ export async function buildDiskImage(build: BootcBuildInfo, history: History): P
       );
       logData += JSON.stringify(buildImageContainer, undefined, 2);
       logData += '\n----------\n';
+      try {
+        fs.writeFileSync(logPath, logData);
+      } catch (e) {
+        // ignore
```

Comment:
would be nice to log as console.debug in case you need to know why it's failing

Reply:
Fixed.